### PR TITLE
switch-lan-play 0.1.0

### PR DESCRIPTION
**Pull request by spacemeowx2**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/switch-lan-play.rb [hunk 1]
```diff
@@ -1,8 +1,10 @@
 class SwitchLanPlay < Formula
   desc "Make you and your friends play games like in a LAN"
   homepage "https://github.com/spacemeowx2/switch-lan-play"
-  url "https://github.com/spacemeowx2/switch-lan-play/archive/v0.0.7.tar.gz"
-  sha256 "0b645fa8bbff18b4b2d1dfd8d975790c6597bc0520c0ab1368387d08f8c2c900"
+  url "https://github.com/spacemeowx2/switch-lan-play.git",
+    :tag      => "v0.1.0",
+    :revision => "eda2c4dcb1db7948fcc0ae5d129fc4b8f4369ca2"
+  head "https://github.com/spacemeowx2/switch-lan-play.git"
```
Comment (javian) [inline Formula/switch-lan-play.rb]: Please remove this line, we don't accept `HEAD` section additions in formula.

**Review by javian — DISMISSED** [review]

**Comment by javian** [comment]
@BrewTestBot test this please

File: Formula/switch-lan-play.rb (hunk 1 above)
Comment (spacemeowx2) [reply Formula/switch-lan-play.rb]: Removed

**Comment by fxcoudert** [comment]
Thanks @spacemeowx2

**Pull request closed without merging**